tests/HTTP/MessageTest.php

```diff
@@ -258,11 +256,15 @@ function testHasHeaders() {
 
     }
 
-    private function createCallback()
+    /**
+     * @param string $content
+     * @return \Closure Returns a callback printing $content to php://output stream
+     */
+    private function createCallback($content)
     {
-        return function() {
+        return function() use ($content) {
             $fd = fopen('php://output', 'r+');
```

Comment:
This is a nitpick and not that important, but instead of opening a stream to `php://output` and writing the contents there, you can just do:

`echo $content;`.

And this will have an identical effect, and possibly even a bit faster. Not important for unittests though.